extensions/kafka-client/runtime/src/main/java/io/smallrye/reactive/kafka/graal/KafkaSubstitutions.java

```diff
@@ -17,6 +26,81 @@ private static String defaultKerberosRealm() throws ClassNotFoundException, NoSu
 
 }
 
+@TargetClass(className = "com.jayway.jsonpath.internal.filter.ValueNodes", innerClass = "JsonNode", onlyWith = HasStrimzi.class)
```

Comment:
This should be in its own substitution class.

Reply:
I think I botched one of my rebases. I'd a separate `StrimziSubstitutions` class.